tests/lib/EventListener/ContentTypeSuggestionsListenerTest.php

```diff
@@ -0,0 +1,182 @@
+<?php
+
+/**
+ * @copyright Copyright (C) Ibexa AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace Ibexa\Tests\AdminUi\EventListener;
+
+use Ibexa\AdminUi\EventListener\ContentTypeSuggestionsListener;
+use Ibexa\AdminUi\Form\Type\Event\ContentCreateContentTypeChoiceLoaderEvent;
+use Ibexa\Contracts\Core\Repository\SearchService;
+use Ibexa\Contracts\Core\Repository\Values\Content\Location;
+use Ibexa\Contracts\Core\Repository\Values\Content\Search\AggregationResult\TermAggregationResult;
+use Ibexa\Contracts\Core\Repository\Values\Content\Search\AggregationResult\TermAggregationResultEntry;
+use Ibexa\Contracts\Core\Repository\Values\Content\Search\AggregationResultCollection;
+use Ibexa\Contracts\Core\Repository\Values\Content\Search\SearchResult;
+use Ibexa\Contracts\Core\Repository\Values\ContentType\ContentType;
+use PHPUnit\Framework\TestCase;
+use Symfony\Component\EventDispatcher\EventDispatcher;
+use Symfony\Contracts\Translation\TranslatorInterface;
+
+final class ContentTypeSuggestionsListenerTest extends TestCase
+{
+    /** @var \Ibexa\Contracts\Core\Repository\SearchService&\PHPUnit\Framework\MockObject\MockObject */
+    private SearchService $searchService;
+
+    /** @var \Symfony\Contracts\Translation\TranslatorInterface&\PHPUnit\Framework\MockObject\MockObject */
+    private TranslatorInterface $translator;
+
+    protected function setUp(): void
+    {
+        $this->searchService = $this->createMock(SearchService::class);
+
+        $this->translator = $this->createMock(TranslatorInterface::class);
+        $this->translator->method('trans')->willReturnArgument(0);
+    }
+
+    public function testSubscribedEvents(): void
+    {
+        $actualSubservients = array_keys(ContentTypeSuggestionsListener::getSubscribedEvents());
+
+        self::assertEquals([
+            ContentCreateContentTypeChoiceLoaderEvent::RESOLVE_CONTENT_TYPES,
+        ], $actualSubservients);
+    }
+
+    public function testSkipSuggestionComputationIfAggregationApiIsNotSupported(): void
```

Comment:
For consistency with e.g. `disableSupportForAggregationAPI`.
```suggestion
    public function testSkipSuggestionComputationIfAggregationAPIIsNotSupported(): void
```